dpctl/tests/test_dparray.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+from dpctl import dparray
+import numpy
+
+
+def func_operation_with_const(dpctl_array):
+    return dpctl_array * 2.0 + 13
+
+
+def multiply_func(np_array, dpcrtl_array):
+    return np_array * dpcrtl_array
+
+
+class TestOverloadList(unittest.TestCase):
+    maxDiff = None
+
+    X = dparray.ndarray((256, 4), dtype='d')
+    X.fill(1.0)
```

Comment:
This variables will be shared among test cases. It means that cases are not independent.